autopush/web/base.py

```diff
@@ -288,27 +312,13 @@ def _router_fail_err(self, fail, router_type=None, vapid=False):
                 self.log.debug(format="Success", status_code=exc.status_code,
                                logged_status=exc.logged_status or 0,
                                client_info=self._client_info)
-                success = True
-                self.metrics.increment('notification.message.success',
-                                       tags=[
-                                           'destination:Direct',
-                                           'router:{}'.format(router_type),
-                                           'vapid:{}'.format(vapid is not None)
-                                       ])
             elif 400 <= exc.status_code < 500:
                 self.log.debug(format="Client error",
                                status_code=exc.status_code,
                                logged_status=exc.logged_status or 0,
                                errno=exc.errno or 0,
                                client_info=self._client_info)
-        if not success:
-            self.metrics.increment('notification.message.error',
-                                   tags=[
-                                        "code:{}".format(exc.status_code),
-                                        "router:{}".format(router_type),
-                                        "vapid:{}".format(vapid is not None)
-                                   ])
-        self._router_response(exc)
+        self._router_response(exc, router_type, vapid)
```

Comment:
Should we pass just the boolean state of VAPID rather than the whole structure? Are we planning on extracting more from the vapid data?